### ISSUE TITLE
[Component Update] Standalone checkbox and radio button rounded corners

### ISSUE THREAD

**Issue by florentmaitre** [Orange-OpenSource/ouds-flutter#437]
[Component Update] Standalone checkbox and radio button should have rounded corners with Sosh and Wireframe themes

The radius can be applied using ouds/💠_control/control-item/border/radius-item-only.

**Comment by Tayebsed93** [comment]
📣 New Firebase App Distribution Alpha upload available 🚀 (Android)

Please, get the new build from App Tester app, or contact the project maintainers (@Tayebsed93) to get access.

Display name: DesignToolbox (alpha) (510)
Version: 0.6.0
Build type: Alpha
Build details (GitHub): 437
cc @B3nz01d (product owner)
cc @Tayebsed93 @nouha06 @AhmedAmineZr (dev team)
cc @jerome-regnier (design team)

**Comment by Tayebsed93** [comment]
📣 New TestFlight Alpha upload available 🚀 (iOS)

Please, get the new build from TestFlight app, or contact the project maintainers (@Tayebsed93) to get access.

Display name: Design Toolbox Flutter (510)
Version: 0.6.0
Build type: Alpha
Build details (GitHub): 437
cc @B3nz01d (product owner)
cc @Tayebsed93 @AhmedAmineZr @nouha06 (dev team)
cc @jerome-regnier (design team)
cc @pya35 (Accessibility)

**Comment by Tayebsed93** [comment]
📣 New Firebase App Distribution Alpha upload available 🚀 (Android)

Please, get the new build from App Tester app, or contact the project maintainers (@Tayebsed93) to get access.

Display name: DesignToolbox (alpha) (517)
Version: 0.6.0
Build type: Alpha
Build details (GitHub): 437
cc @B3nz01d (product owner)
cc @Tayebsed93 @nouha06 @AhmedAmineZr (dev team)
cc @jerome-regnier (design team)

**Comment by Tayebsed93** [comment]
📣 New TestFlight Alpha upload available 🚀 (iOS)

Please, get the new build from TestFlight app, or contact the project maintainers (@Tayebsed93) to get access.

Display name: Design Toolbox Flutter (517)
Version: 0.6.0
Build type: Alpha
Build details (GitHub): 437
cc @B3nz01d (product owner)
cc @Tayebsed93 @AhmedAmineZr @nouha06 (dev team)
cc @jerome-regnier (design team)
cc @pya35 (Accessibility)

**Comment by jerome-regnier** [comment]
ok Design pour Sosh et Wireframe pour le radius corner des interactions

**Comment by Tayebsed93** [comment]
closed with PR #490

**Comment by B3nz01d** [comment]
Finished during Sprint # 25
